tests/manual/controls/SelectTableViewTest.java

```diff
@@ -0,0 +1,112 @@
+import javafx.application.Application;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.collections.ObservableList;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.SelectionMode;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+public class SelectTableViewTest extends Application {
+
+    final int COL_COUNT = 3;
+    final int ROW_COUNT = 70_000;
+    //  final int ROW_COUNT = 80_000;
+    //  final int ROW_COUNT = 50_000;
+    //  final int ROW_COUNT = 8_000;
```

Comment:
The number is meaningful because it is the number of data I used to show the improvement effect in the original PR.

Reference value of the number of data that can be handled within 3 seconds of processing time (before-> after)

TableView

selectAll: 8_000-> 700_000
selectRange: 7_000-> 50_000





``` Java
public class SelectTableViewTest extends Application {

	final int ROW_COUNT = 700_000;
//	final int ROW_COUNT = 80_000;
//	final int ROW_COUNT = 50_000;
//	final int ROW_COUNT = 8_000;
	final int COL_COUNT = 3;

```

Reply:
I reduced ROW_COUNT from 700_000 to 70_000 as the tests were taking a few seconds to run on my machine. I have reverted these now. Nevertheless, time taken to run a test have a number of variables. Depending on the machine the tests are run, it may not necessarily always take 3 seconds :)

Reply:
The processing time changes for each operation.
You probably adapted to the slow operation (selectRange).
You can comment it out, but 700_000 is for selectAll. Although it depends on the device, the environment I have confirmed is his MacBook Pro in 2016.